checks.lua

```diff
@@ -226,17 +230,19 @@ function checkers.int64(arg)
     return false
 end
 
-checkers.tuple = box.tuple.is
-
 local has_decimal, decimal = pcall(require, 'decimal')
 if has_decimal and decimal.is_decimal then
     checkers.decimal = decimal.is_decimal
 end
 
--- https://github.com/tarantool/tarantool/blob/7682d34162be34648172d91008e9185301bce8f6/src/lua/uuid.lua#L29
-local uuid_t = ffi.typeof('struct tt_uuid')
-function checkers.uuid(arg)
-    return ffi.istype(uuid_t, arg)
+if is_tarantool() == true then
```

Comment:
If the idea is to move all tatantool specific checkers under this if statement, I would also move decimal here. Or define `has_<...>` variables for tuple and uuid. Now we use different approaches here and there.

(As for me, check, whether a feature is present is better than check whether we're on tarantool.)

Reply:
Yes, I know that decimal is a tarantool-specific data type. But current approach works fine for support luajit and tarantool and thus fixes a problem.

Reply:
Looks like `if is_tarantool() then ...` should work.